database/migrations/2024_03_18_164714_add_decline_msg_to_checkout_acceptance_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddDeclineMsgToCheckoutAcceptanceTable extends Migration
```

Comment:
Small change but helpful: can we update this class and file name to remove the `decline` reference?